gammapy/modeling/models/prior.py

```diff
@@ -0,0 +1,149 @@
+import logging
+import numpy as np
+import astropy.units as u
+from gammapy.modeling import PriorParameter, PriorParameters
+from .core import ModelBase
+
+log = logging.getLogger(__name__)
+
+
+def _build_priorparameters_from_dict(data, default_parameters):
+    """Build Parameters object from input dict and default  parameter values."""
```

Comment:
Maybe one can describe in the docstring the parameters, even for the developers...